src/style.css

```diff
@@ -0,0 +1,96 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: bold;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  --size-big: 70px;
+  --size-small: 42px;
+}
+
+.search-bar {
+  position: relative;
+}
+
+.search-bar-big {
+  /*position: relative -> all indentations setup by top, right, bottom, left*/
+  top: 12px;
+}
+
+.search-bar-small {
+  /*position: relative -> all indentations setup by top, right, bottom, left*/
+  top: 32px;
+}
```

Comment:
as I wrote earlier it's better to omit using relative position if you can achieve the same result with margins